samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -127,6 +127,7 @@
    */
   private static final String AM_JMX_ENABLED = "yarn.am.jmx.enabled";
   private static final String CLUSTER_MANAGER_JMX_ENABLED = "cluster-manager.jobcoordinator.jmx.enabled";
+  private static final String CLUSTER_MANAGER_STARTPOINT_FANOUT_ENABLED = "job.startpoint.fanout.enabled";
```

Comment:
If this config is called `job.startpoint.fanout.enabled`, isn't it better to be a part of JobConfig than ClusterManagerConfig?

Reply:
Once you decide where it fits, since this is user-facing, please add docs to doc markdown. Thanks!